app/_hub/kong-inc/oauth2-introspection/index.md

```diff
@@ -46,88 +47,101 @@ params:
       required: true
       default:
       value_in_examples: https://example-url.com
+      datatype: string
       description: |
-        The full URL to the third-party introspection endpoint
+        The full URL to the third-party introspection endpoint.
     - name: authorization_value
       required: true
       default:
       value_in_examples: Basic MG9hNWlpbjpPcGVuU2VzYW1l
+      datatype: string
       description: |
         The value to set as the `Authorization` header when querying the introspection endpoint. This depends on the OAuth 2.0 server, but usually is the `client_id` and `client_secret` as a Base64-encoded Basic Auth string (`Basic MG9hNWl...`).
     - name: token_type_hint
       required: false
       default:
       value_in_examples:
+      datatype: string
       description: |
-        The token_type_hint value to associate to introspection requests
+        The `token_type_hint` value to associate to introspection requests.
     - name: ttl
       required: false
-      default: 60
+      default: 30
       value_in_examples:
+      datatype: number
       description: |
-        The TTL in seconds for the introspection response - set to 0 to disable the expiration
+        The TTL in seconds for the introspection response. Set to 0 to disable the expiration.
     - name: hide_credentials
       required: false
       default:
       value_in_examples:
+      datatype: boolean
       description: |
         An optional boolean value telling the plugin to hide the credential to the upstream API server. It will be removed by Kong before proxying the request.
     - name: timeout
       required: false
       default: 10000
       value_in_examples:
+      datatype: integer
       description: |
-        An optional timeout in milliseconds when sending data to the upstream server
+        An optional timeout in milliseconds when sending data to the upstream server.
     - name: keepalive
       required: false
       default: 60000
       value_in_examples:
+      datatype: integer
       description: |
-        An optional value in milliseconds that defines for how long an idle connection will live before being closed
+        An optional value in milliseconds that defines how long an idle connection lives before being closed.
     - name: anonymous
       required: false
       default:
       value_in_examples:
+      datatype: string
       description: |
         An optional string (consumer uuid) value to use as an "anonymous" consumer if authentication fails. If empty (default), the request will fail with an authentication failure 4xx.
     - name: run_on_preflight
       required: false
```

Comment:
```suggestion
      required: true
```

Reply:
Just wanted to point out that it's a required field. Feel free to ignore this suggestion if you don't want to scope creep.

Reply:
This isn't one that had its schema explicitly updated for required booleans:

  { run_on_preflight = { type = "boolean", default = true } },